crates/viewer/re_data_ui/src/instance_path.rs

```diff
@@ -59,106 +70,222 @@ impl DataUi for InstancePath {
                     if indicator_count > 1 { "s" } else { "" }
                 ),
             );
-            return;
+        } else {
+            component_list_ui(
+                ctx,
+                ui,
+                ui_layout,
+                query,
+                db,
+                entity_path,
+                instance,
+                &components,
+            );
         }
 
-        let show_indicator_comps = match ui_layout {
-            UiLayout::Tooltip => {
-                // Skip indicator components in hover ui (unless there are no other
-                // types of components).
-                indicator_count == components.len()
+        if instance.is_all() {
+            let component_map = components.into_iter().collect();
+            preview_if_image_ui(ctx, ui, ui_layout, query, entity_path, &component_map);
```

Comment:
I wish there was a more scalable way for "pan entity ui" already. But this simple special handling will probably go a long way :shrug: